### ISSUE TITLE
Rename `kwip-entvec` something more sane

### ISSUE THREAD

**Issue by kdmurray91** [kdmurray91/kWIP#7]
Probably `kwip-stats`.

**Comment by kdmurray91** [comment]
@warthmann did you have any suggestion here?

**Comment by warthmann** [comment]
Is it unreasonable that entvec becomes a flag of kwip? I currently always run both as I want the entvec output. Better to run kwip only once with additional flag.

Norman

Sent from a phone

On 10.01.2016, at 17:28, Kevin Murray <notifications@github.com<mailto:notifications@github.com>> wrote:


@warthmann<https://github.com/warthmann> did you have any suggestion here?

-

**Comment by kdmurray91** [comment]
So those stats are calculated every time kWIP is run (or at least turned on with a CLI flag)? It's not as simple as flicking a switch, but should be possible. It would have to be part of a refactor to allow writing out the entropy vector, population hash and population frequency tables.

For now, I think I'll rename this command `kwip-stats` and create a separate issue about the above refactoring to be dealt with post-paper.

**Comment by warthmann** [comment]
Perfect!

Norman

Sent from a phone

On 10.01.2016, at 20:22, Kevin Murray <notifications@github.com<mailto:notifications@github.com>> wrote:


So those stats are calculated every time kWIP is run (or at least turned on with a CLI flag)? It's not as simple as flicking a switch, but should be possible. It would have to be part of a refactor to allow writing out the entropy vector, population hash and population frequency tables.

For now, I think I'll rename this command kwip-stats and create a separate issue about the above refactoring to be dealt with post-paper.

-

**Comment by kdmurray91** [comment]
Done in be41177
